src/components/DialogBody.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import CloseIcon from './CloseIcon';
+
+const DialogBody = ({ locationDetails, onClick }) => (
```

Comment:
Love that you separated this out into its own component. You might even give this a specific name like `LocationDetailsModalBody` (that's kind of wordy! 😄) since it's for a specific purpose.

Reply:
I know I am all for clarity even if it means a couple of extra words!